 AudioFlinger: MR0_AUDIO_BLOB: fix AudioRecord

Our old audio library does not support hardware timestamps
and return an incorrect value. This causes libaudioflinger crashes when
attempting to record audio.

diff --git a/services/audioflinger/Threads.cpp b/services/audioflinger/Threads.cpp
index 0caada8..89e2084 100644
--- a/services/audioflinger/Threads.cpp
+++ b/services/audioflinger/Threads.cpp
@@ -6292,6 +6292,7 @@ reacquire_wakelock:
         mTimestamp.mPosition[ExtendedTimestamp::LOCATION_SERVER] += framesRead;
         mTimestamp.mTimeNs[ExtendedTimestamp::LOCATION_SERVER] = systemTime();
 
+#ifndef MR0_AUDIO_BLOB
         // Update server timestamp with kernel stats
         if (mInput->stream->get_capture_position != nullptr
                 && mPipeSource.get() == nullptr /* don't obtain for FastCapture, could block */) {
@@ -6307,6 +6308,7 @@ reacquire_wakelock:
                 // as the read obtains a lock, preventing the timestamp call from executing.
             }
         }
+#endif
         // Use this to track timestamp information
         // ALOGD("%s", mTimestamp.toString().c_str());
 
